### PR TITLE
Fix push replication, Reactivate service

### PR DESCRIPTION
**Pull request by jacquarg**
- Reactivate service and update its init states.
- Continue the work of @kosssi in #170 , and fix bidirectionnal filtered replication
- Handle conflicts (for any docTypes)

### CODE REVIEW

File: www-src/app/replicator/replication_launcher.coffee [hunk 1]
```diff
@@ -88,6 +90,7 @@ module.exports = class ReplicationLauncher
         if options.live
             liveOptions =
               retry: true
+              heartbeat: false
```
Comment (nono) [inline www-src/app/replicator/replication_launcher.coffee]: Why no heartbeat?

File: www-src/app/replicator/replicator_backups.coffee [hunk 2]
```diff
@@ -104,7 +104,19 @@ module.exports =
                 else
                     callback()
 
-
+    # Uupload photos take with device on the cozy.
```
Comment (nono) [inline www-src/app/replicator/replicator_backups.coffee]: s/Uupload/Upload/

**Comment by nono** [comment]
Looks good to me

File: www-src/app/replicator/replication_launcher.coffee (hunk 1 above)
Comment (jacquarg) [inline www-src/app/replicator/replication_launcher.coffee]: It's still broken (PouchDB nunkown error) when I debug with my cozy-dev . So I removed heartbeat to remove noise while debuggin, but I haven't yet try with a prod cozy, to check if it works as well as kossi experience it.
Just a minute and I will remove it from this PR : )

**Pull request merged**